Dockerfile

```diff
@@ -41,12 +41,13 @@ RUN apt-get -y autoremove && apt-get clean && apt-get autoclean && \
 RUN mkdir -p /var/talis-php
 COPY . /var/talis-php
 
-RUN echo 'export PERSONA_TEST_HOST=https://staging-users.talis.com' >> /root/.bashrc
-RUN echo 'export PERSONA_TEST_OAUTH_CLIENT='$persona_oauth_client >> /root/.bashrc
-RUN echo 'export PERSONA_TEST_OAUTH_SECRET='$persona_oauth_secret >> /root/.bashrc
+RUN echo "export PERSONA_TEST_HOST='https://staging-users.talis.com'" >> /etc/profile.d/test.sh \
```

Comment:
.bashrc isn't executed for non interactive shells. Moved to `/etc/profile.d/test.sh` to be able to call it.